support-frontend/assets/helpers/tracking/ccpa.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+// ----- Imports ----- //
+
+import {
+  detect as detectCountry,
+  type IsoCountry,
+} from 'helpers/internationalisation/country';
+
+// ----- Functions ----- //
+
+export const ccpaEnabled = (): boolean => {
+  const useCCPA = true; // set false to switch CCPA off
```

Comment:
Is it worth having this in the switchboard?

Reply:
Yes definitely, this was my intention which I listed as a todo, I'm not sure if/how we uses switches on support-frontend, any guidance/examples would be appreciated.